examples/publish-ci/react-native/src/constants/TypedColors.ts

```diff
@@ -0,0 +1,13 @@
+import { Colors } from "react-native/Libraries/NewAppScreen"
+
+interface AllColors {
+  primary: string
+  white: string
+  lighter: string
+  light: string
+  dark: string
+  darker: string
+  black: string
+}
+
+export const TypedColors = Colors as AllColors
```

Comment:
out of curiosity, why not just add a declaration for the original Colors variable?

Reply:
Oh I just remembered, it wasn't missing a declaration, the `Colors` variable was just typed as `any`. And this was my way of overriding it to make it more type safe. Is there a better a way to accomplish that without doing what I did there?

Reply:
you're already providing types for `"react-native/Libraries/NewAppScreen"` in `globals.d.ts`, just add the Colors export to it 

```diff
declare module "react-native/Libraries/NewAppScreen" {
  import type { FC } from "react"
  export const HermesBadge: FC
+
+  export const Colors: {
+    primary: string;
+    white: string;
+    lighter: string;
+    light: string;
+    dark: string;
+    darker: string;
+    black: string;
+  };
}
```

Reply:
@EskiMojo14 I get `Cannot redeclare block-scoped variable 'Colors'.ts(2451) LaunchScreen.d.ts(15, 16): 'Colors' was also declared here.`

Reply:
ah, interesting - apparently my dependencies hadn't actually installed 

then yeah, i guess this is necessary - shame the original types are so rubbish